### PR TITLE
[Merged by Bors] - Update cli smartmodule

### PR DESCRIPTION
**Pull request by crajcan**
https://github.com/infinyon/fluvio-website/issues/553

Dependent on: https://github.com/infinyon/fluvio/pull/3177

### CODE REVIEW

**Review by tjtelan — APPROVED** [review]
Looks good! I'll merge this when the related PR in Fluvio merged

File: content/cli/commands/produce.md [hunk 1]
```diff
@@ -62,6 +62,26 @@ Ok!
 
 The producer splits the key from the value and prints it in a `[key] value` format.
 
+### Produce using a SmartModule
```
Comment (ajhunyady) [inline content/cli/commands/produce.md:65]: Question: Is `--smartmodule-path` the only way to produce with smartmodules? This is great for testing, but not ideal when multiple clients want to call the same smartmodule when producing. Is there also a command for the producer to use a smartmodule that has already been uploaded?

Comment (ajhunyady) [reply content/cli/commands/produce.md:65]: That's great! It may be best to add a 2nd paragraph, that way we cover both cases.

**Review by ajhunyady — APPROVED** [review]
This is great! Thank you!

**Pull request closed without merging**